website/package.json

```diff
@@ -3,22 +3,37 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "docusaurus-start",
-    "build": "docusaurus-build",
-    "gh-pages": "docusaurus-publish",
-    "examples": "docusaurus-examples",
-    "crowdin-upload": "crowdin --config ../crowdin.yaml upload sources --auto-update -b master",
-    "crowdin-download": "crowdin --config ../crowdin.yaml download -b master",
-    "publish-gh-pages": "docusaurus-publish",
-    "write-translations": "docusaurus-write-translations",
-    "version": "docusaurus-version",
-    "rename-version": "docusaurus-rename-version"
+    "docusaurus": "docusaurus",
```

Comment:
does nothing, does it?

Reply:
It is useful for npm users in case we add a new script that you forget to add to your package.json.

You can run and subcommand of our cli using it, like `npm run docusaurus the-new-shiny-docusaurus-command -- --option1`

It's added in the default Docusaurus 2 template btw, it's always convenient to have.

Note: yarn does not need it to be able to run `yarn docusaurus the-new-shiny-docusaurus-command --option1`, it's really for npm users, so if you find it useless for this specific site it can probably be removed, but it wouldn't harm much to keep it ^^

Reply:
This repo uses Yarn so npm shouldn't be used at all as they are not compatible

Reply:
Removed it. 
Just pointing out that a newly initialized Docusaurus site always has this script, just wanted the scripts of Jest website to look similar to any other docusaurus site, as site authors most likely don't remove that script, even when using yarn.